core-bundle/src/Resources/config/listener.yml

```diff
@@ -16,6 +16,13 @@ services:
         tags:
             - { name: kernel.event_listener, event: kernel.terminate, method: onKernelTerminate }
 
+    contao.listener.robots_txt:
```

Comment:
We have agreed on using the class name as service key for all new services, so this needs to become `Contao\CoreBundle\EventListener\RobotsTxtListener`.